FinniversKit/Sources/Components/Ribbon/IconRibbonView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
```

Comment:
This doesn't seem to be compatible with the top-level LICENSE, nor is FINN.no an Inc.